Products/zms/__init__.py

```diff
@@ -252,6 +252,18 @@ def initialize(context):
                 standard.writeStdout(context, "add %s (Packed: %i -> %i Bytes)"%(fn, l0, l1))
               fileobj.write(fc)
             fileobj.close()
+            # Finally create hash value of packed file content:
+            # a. write it into separate file and
+            # b. create a configuration parameter
+            min_fileobj = open(fileobj.name, 'r')
+            min_hash = standard.encrypt_password(min_fileobj.read(),hex=True)
```

Comment:
Why don't you use the python builtin hash()-function and hexlify the result?
Password encryption maybe misleading and change its behaviour.

Reply:
the ZMS-builtin function does exactly the required task:
https://github.com/zms-[publishing/ZMS/blob/f5d6275e83ab7d63e33ccf1a03ed2b5ce0997917/Products/zms/conf/metaobj_manager/captcha_lib/captcha_func.py#L46-L57](https://github.com/zms-publishing/ZMS/blob/f5d6275e83ab7d63e33ccf1a03ed2b5ce0997917/Products/zms/standard.py#L622-L645)

Of course the function's name might suggest a special purpose and a limited use. So feel free to apply the python-builtin.